src/couch_mrview_http.erl

```diff
@@ -360,29 +360,34 @@ view_cb({meta, Meta}, #vacc{}=Acc) ->
     end ++ ["\"rows\":["],
     Chunk = [prepend_val(Acc), "{", string:join(Parts, ","), "\r\n"],
     {ok, AccOut} = maybe_flush_response(Acc, Chunk, iolist_size(Chunk)),
-    {ok, AccOut#vacc{prepend=""}};
+    {ok, AccOut#vacc{prepend="", meta_sent=true}};
 view_cb({row, Row}, #vacc{resp=undefined}=Acc) ->
-    % sorted=false and a row arrived before meta, start response.
+    % sorted=false and a row arrived before meta, and response hasn't started
     Pre = "{\"rows\":[\r\n",
     {ok, Resp} = chttpd:start_delayed_json_response(Acc#vacc.req, 200, []),
-    view_cb({row, Row}, Acc#vacc{row_sent=true,resp=Resp, prepend=Pre, should_close=true});
+    view_cb({row, Row}, Acc#vacc{resp=Resp, row_sent=true, prepend=Pre, should_close=true});
```

Comment:
it seems you just reordered this? For clarity, I think you should not do so, as it makes everyone stare at this line for a real change (is there one??)